tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/tuples/BasicTupleTest.java

```diff
@@ -325,6 +328,22 @@ public void testTupleAsTupleFirstMember() {
         BRunUtil.invoke(result, "testTupleAsTupleFirstMember");
     }
 
+    @Test
+    public void testTupleName() {
+        Object returns = BRunUtil.invoke(result, "testTupleName", new Object[]{});
+        TupleValueImpl returnTuple = (TupleValueImpl) returns;
+
+        Type t1 = TypeUtils.getType(returnTuple.get(0));
+        Type t2 = TypeUtils.getType(returnTuple.get(1));
+        Type t3 = TypeUtils.getType(returnTuple.get(2));
+        Type t4 = TypeUtils.getType(returnTuple.get(3));
```

Comment:
Do we need these separate variables? Even if we are continuing with them can we move them closer to usage?